### PR TITLE
Integrate desktop version by media queries

### PR DESCRIPTION
**Pull request by evillaz**
Integrated media queries to adapt some flex containers, grid templates, grid item positioning, and styling to make the project fit the Figma template for the desktop version
created new elements to display instead of some mobile ones.
Applied transformations made in today's pair programming session.
*Fixed width on reviewed element.
*fixed applying flex properties for responsive resizing
*Fixed footer element only showing on mobile version

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -81,17 +65,373 @@ ul {
...
+    flex-direction: row;
+    gap: 24px;
+    padding: 130px 142px 154px 135px;
+  }
+
+  #contactDescription {
+    width: 31.6%;
+    font-size: 40px;
+    line-height: 130%;
+  }
```
Comment (franclobo) [inline css/style.css]: - [ ] Kindly correct the width of the contact description t be similar to the Figma design. 👇🏽 

| Figma design | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @evillaz, 

# ♻️ Required Changes ♻️

soClose

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -352,40 +476,40 @@ <h4 class="featureText">Terminal</h4>
...
-        <form class="flex-container" action="https://formspree.io/f/xnqyndrw" method="POST">
+        <p id="contactDescription">I'm always interested <br>in hearing about new <br>projects, so if you'd <br>like to chat please <br>get in touch.</p>
+        <form class="flex-container" id="contactForm" action="https://formspree.io/f/xnqyndrw" method="POST">
           <input class="formElement" type="text" name="name" placeholder="Full name" maxlength="30" required>
           <input class="formElement" type="email" name="email" placeholder="Email address" required>
           <textarea class="formElement name="message" placeholder="Enter text here" maxlength="500" required></textarea>
           <button class="button" id="submitForm" type="submit">Get in touch</button>
         </form>
       </section>
     </div>
```
Comment (Ibizugbe) [inline index.html]: [REQUIRED]As requested by the previous reviewer, I think your project still looks a bit different from the design😉 don't forget it is a requirement that we stick to the design even as we make our page responsive😉 kindly see the guiding lines on the image and make the changes🤗

YOUR APP
your___________

DESIGN
design_________

[OPTIONAL] I think the thick black line highlighted with yellow shouldn't be a part of your application😉 That is just a navigation usually present in mobile phones😉 probably the designer uses that to have a better illustration of the look of the application on mobile devices😉 I suggest you take it away as it makes your app look less professional😉

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @evillaz 👋, 

68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅
- linters are passing👍
- readme looks professional👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @evillaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**Kindly check an optional comment I left under the review**_ :pray: 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  You wrote a good README file but not professional. :heavy_check_mark: 
-  Correct GitHub flow. :heavy_check_mark: 



## Optional suggestions

- [OPTIONAL] Part of general requirements is to remove the Microverse logo and optional add your logo if you have one. Kindly remove the Microverse logo on your README file. 

Your-readme


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**